README.md

```diff
@@ -5,6 +5,25 @@ By storing them in a central place (here), and distributing them in an automated
 1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
 2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
 
+## Customization
+
+Most repositories won't need any customization, and the workflows defined here will just work fine.
+Some repositories may require some pre-setup steps to be run before tests (or code checks) can be run. Setup steps for `go-test` are defined in `.github/actions/go-test_setup/action.yml`, and setup steps for `go-check` are defined in `.github/actions/go-check_setup/action.yml`, in the following format:
```

Comment:
I imagine a fairly common use case will be installing extra build dependencies, like for cgo, in which case the same step will be required for both. I guess we're fine with a bit of copy pasting or symlinking for that use case?

Reply:
I think we are.

Another common case would be installing and running a background job (IPFS daemon), which we'll only need for `go-test`, which I'd like to avoid for `go-check`.